sd_webui_bayesian_merger/tpe_optimiser.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+from sd_webui_bayesian_merger.artist import draw_unet
+from sd_webui_bayesian_merger.merger import NUM_TOTAL_BLOCKS
+from sd_webui_bayesian_merger.optimiser import Optimiser, convergence_plot
+
+from hyperopt import Trials, hp, fmin, tpe, STATUS_OK
+
+
+class TPEOptimiser(Optimiser):
+
+    def _target_function(self, params):
+        res = self.sd_target_function(**params)
+        return {
+            'loss': -res,
```

Comment:
`-res` or `1/(res + 1e-10)` ?

Reply:
We want to maximise the value don't we? At least that's what I gathered from the Bayesian Optimiser usage

Reply:
yes indeed

I was wondering if there's a difference for tpe in minimizing a negative number towards -infinite or a number between 0 and 1

probably it doesn't matter